docs/operations.md

```diff
@@ -28,6 +28,138 @@ Clients may inspect this specification at run-time so to obey the various limits
+2. Upon receipt of the request, `wfx` sets the `Content-Type` header to `text/event-stream`.
+3. The server then initiates a stream of job events in the response body, allowing clients to receive instant updates.
+
+#### Event Format Specification
+
+The event stream looks as follows:
+
+```
+data: [...]
+id: 1
+
+data: [...]
+id: 2
+
+[...]
+```
+
+with a single event being structured as follows:
+
+```
+data: { "action": "<ACTION>", "job": <JOB> }
+id: <EVENT_ID>
+
+```
+
+**Note**: Each event is terminated by a pair of newline characters `\n\n` (as required by the SSE spec).
+
+The semantics of the individual fields is:
+
+- `<ACTION>` specifies the type of event that occurred. The valid actions are:
+  - `CREATE`: a new job has been created
+  - `DELETE`: an existing job has been deleted
+  - `ADD_TAGS`: tags were added to a job
+  - `DELETE_TAGS`: tags were removed from a job
+  - `UPDATE_STATUS`: job status has been updated
+  - `UPDATE_DEFINITION`: job definition has been updated
+- `<JOB>` is the JSON representation of the job associated with this event
+- `<EVENT_ID>`: a (unique) numeric identifier for the event, starting at 1 and incremented by 1 for each subsequent event; can be used to detect message loss on the client side.
+
+**Example:**
+
+```
+data: {"action":"UPDATE_STATUS","job":{"clientId":"Dana","definition":{"title":"Expose Job API"},"history":null,"id":"d305e539-1d41-4c95-b19a-2a7055c469d0","mtime":"2023-08-29T13:00:57.632+02:00","status":{"definitionHash":"f8b3d6ea0150a98ec5f8dc224659211a37cb4737777388f77a3e323a865e7674","state":"PROGRESS"},"stime":"2023-08-22T15:30:02.323+02:00","tags":[],"workflow":{"groups":null,"name":"wfx.workflow.kanban","states":null,"transitions":null}}}
+id: 1
+
+```
+
+#### Filter Parameters
+
+Job events can be filtered using any combination of the following parameters:
+
+- Job IDs
+- Client IDs
+- Workflow Names
+
+This enables more precise control over the events which are dispatched.
+
+#### Examples
+
+`wfxctl` offers a reference client implementation. The following command subscribes to **all** job events:
+
+```bash
+wfxctl job events
+```
+
+This may result in a large number of events, though. For a more targeted approach, filter parameters may be used.
+Assuming the job IDs are known (either because the jobs have been created already or the IDs are received via another
+subscription channel), the following will subscribe to events matching either of the two specified job IDs:
+
+```bash
+wfxctl job events --ids d305e539-1d41-4c95-b19a-2a7055c469d0,e692ad92-45e6-4164-b3fd-8c6aa884011c
+```
+
+See `wfxctl job events --help` for other filter parameters, e.g. workflow names.
+
+#### Considerations and Limitations
+
+1. **Asynchronous Job Status Updates**: Job status updates are sent asynchronously to avoid the risk of a subscriber
+   causing delays. As a result, messages could arrive in a not necessarily linear order, deviating from the client's
```

Comment:
Hm, "[...] a subscriber causing message delivery stalls" would be more descriptive?